src/Components/Guests.jsx

```diff
@@ -26,7 +26,7 @@ class Guests extends React.Component{
             <Form onClick={this.props.changeVisibility}>
         <GuestCount> {adults + children} {adults + children === 1 
         ? "guest" : "guests"}{infants === 0 ? '' : `, ${infants} ${infants === 1 
-            ? "infant" : "infants"}`}</GuestCount><DropDown>\/</DropDown>
+            ? "infant" : "infants"}`}</GuestCount><DropDown><svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" clip-rule="evenodd"><path d="M23.245 4l-11.245 14.374-11.219-14.374-.781.619 12 15.381 12-15.391-.755-.609z"/></svg></DropDown>
```

Comment:
I see that you surrendered and used svg in the end.

Reply:
Thanks for giving a tip. I tried and it indeed looks better